README.md

```diff
@@ -50,9 +50,11 @@ Activate the module by adding it to the config.js file as shown in the example b
 | odometer        | Total mileage of the vehicle (rounded to 2 decimal places)                                  |
 | temperature     | Temperature outside and inside the vehicle (see note below)                                 |
 | data-time       | How long ago the data was collected by TeslaFi                                              |
+| state           | Vehicle State (Idling/Driving/Sentry) (see note below)                                      |
 
 - Some fields (charge-time, charge-added) are only enabled if the vehicle is plugged in
 - The temperature field may not be populated if you use TeslaFi's sleep mode, which will stop this row from showing entirely. You may need to use `apiQuery: "&command=lastGoodTemp"` if this fails to show
+- State field will display vehicle's speed (in preferred units) and vehicle's heading when the state is "Driving"
```

Comment:
I think we can probably fit this in to the main table rather than having a separate comment, especially if we split `speed` and `heading` as in the comment above

Reply:
Agreed, will split it out.
This was the first one I built, hadn't thought that far ahead. 
On it!